### ISSUE TITLE
Crashes when there is no element or one element

### ISSUE THREAD

**Issue by trauzti** [TonicArtos/SuperSLiM#11]
I love this library and the ability to switch from 3 elements per row to 1 element dynamically.
However, when there is zero elements or no elements in the array adapter I get a nullpointerexception:

    java.lang.NullPointerException: Attempt to invoke virtual method
    'android.view.ViewGroup$LayoutParams android.view.View.getLayoutParams()'
    on a null object reference

Is there any workaround for this or is it simply a bug?

**Comment by zjuhasz** [comment]
I am experiencing this as well. I did a test where I took the example project and set getItemCount in CountryNamesAdapter to 0 and got the same bug. It's weird because I looked into a bit and there seems to be logic in SuperSLiM to check if the size is greater than 0.

I'm going to move on to another part of my project and hope that someone comments on this, if not I'll look into it more a couple weeks. I'd like to say it looks like a really cool library though, lots of potential and it fills a big gap in Android layouts. Thanks for all the work that went into this.
